test/webserver.mjs

```diff
@@ -158,7 +163,7 @@ WebServer.prototype = {
         return;
       }
       if (isDir) {
-        serveDirectoryIndex(filePath);
+        await self.#serveDirectoryIndex(res, pathPart, queryPart, filePath);
```

Comment:
Do we actually need to await this call (and thus make `statFile` async), since we're not actually returning anything from this method?
Even before this patch the function was asynchronous, but in a different way, since `fs.readdir` used a callback function.

Reply:
I have checked and it's indeed not necessary, so I have removed it in the new commit. Thanks!